components/builder-originsrv/src/migrations/origin_projects.rs

```diff
@@ -446,5 +446,76 @@ pub fn migrate(migrator: &mut Migrator) -> SrvResult<()> {
                      END
                  $$ LANGUAGE plpgsql VOLATILE"#,
     )?;
+    migrator.migrate(
+        "originsrv",
+        r#"CREATE TABLE IF NOT EXISTS project_integration(
+            project_id bigint REFERENCES origin_projects(id) ON DELETE CASCADE,
+            integration_id bigint REFERENCES origin_integrations(id) ON DELETE CASCADE,
+            PRIMARY KEY (project_id, integration_id)
+        );"#,
+    )?;
+    // Normally I would guard on this running every time but the table we're migrating from gets created above this.
+    // So we will get no results here after the first pass and never insert again
+    // Forever alone
+    migrator.migrate(
+        "originsrv",
+        r#"INSERT INTO project_integration (project_id, integration_id)
+            SELECT op.id as project_id, oi.id as integrations_id
+            FROM origin_project_integrations opi
+            JOIN origin_projects op ON opi.name = op.package_name
+            JOIN origin_integrations as oi on opi.integration = oi.name;"#,
+    )?;
+    migrator.migrate(
+        "originsrv",
+        "DROP TABLE IF EXISTS origin_project_integrations CASCADE",
```

Comment:
In think you want to hold off on this drop until the next release. This is akin to deleting a v1 proc since once this table is gone you have broken compatibility.